### PR TITLE
NFDIV-843: Event to enable Amed case

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###

NFDIV-843

### Change description ###
 Event to enable Amed case

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            .event(CASEWORKER_AMEND_CASE)
+            .forStates(AwaitingAos, AosDrafted, AosOverdue,
+                DefendedDivorce, Holding, AwaitingConditionalOrder,
+                ConditionalOrderDrafted, AwaitingLegalAdvisorReferral, AwaitingClarification,
+                ConditionalOrderRefused, AwaitingPronouncement, ConditionalOrderPronounced)
+            .name("Update Case")
+            .description("Update Case")
+            .explicitGrants()
+            .grant(CREATE_READ_UPDATE,
+                CASEWORKER_COURTADMIN_CTSC)
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java:54]: do we also need `CASEWORKER_COURTADMIN_RDU` ?

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class AmendCase implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder.page("amendCase")
+            .complex(CaseData::getApplication)
+                .complex(Application::getMarriageDetails)
+                    .optional(MarriageDetails::getDate)
+                    .optional(MarriageDetails::getPlaceOfMarriage)
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java:17]: do we need this as optional or mandatory ? as there is a both are mandatory I think when we create application

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java (hunk 1 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java:54]: Story states only about - caseworker-courtadmin

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java (hunk 2 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java:17]: Just followed Divorce - 1 configuration.

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java (hunk 1 above)
Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAmendCase.java:54]: court admin CTSC and courtadmin RDU ?

**Review by nitinprabhu — APPROVED** [review]

**Pull request closed without merging**